store/modules/events.ts

```diff
@@ -12,22 +30,14 @@ type CommunityEvent = {
   to: string
 }
 
-type CommunityEventTypes = {
-  [key: string]: CommunityEventType
-}
-
-const COMMUNITY_EVENT_TYPES: CommunityEventTypes = {
-  hackathon: 'Hackathon',
-  camp: 'Camp',
-  unconference: 'Unconference',
-  industryEvent: 'Industry Event',
-  workshop: 'Workshop',
-  talks: 'Talks',
-  virtualEvent: 'Virtual Event'
-}
-
-const LOCATION_CATEGORIES: WorldLocation[] = ['Americas', 'Asia Pacific', 'Europe', 'Africa', 'Online']
-const TYPE_CATEGORIES: CommunityEventType[] = [
+const WORLD_LOCATION_OPTIONS: WorldLocation[] = [
+  WORLD_LOCATIONS.americas,
+  WORLD_LOCATIONS.asiaPacific,
+  WORLD_LOCATIONS.europe,
+  WORLD_LOCATIONS.africa,
+  WORLD_LOCATIONS.online
+]
+const TYPE_OPTIONS: CommunityEventType[] = [
```

Comment:
I like the correspondence `WORLD_LOCATION_OPTIONS` => `WorldLocation`. Could you use the same for `TYPE_OPTIONS` replacing it with `COMMUNITY_EVENT_TYPE_OPTIONS` (=> `CommunityEventType`)??